types/decentlab/PR-26/default.schema.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$id": "https://akenza.io/decentlab/pr-26-default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Default",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "https://raw.githubusercontent.com/akenza-io/device-type-library/master/data-models/ambiance/temperature/schema.json"
+    },
+    {
+      "properties": {
+        "pressure": {
+          "title": "Pressure",
+          "unit": "bar",
+          "description": "Barometric Pressure in bar",
+          "type": "number",
+          "minimum": -0.1,
+          "maximum": 1
```

Comment:
are you sure about these?

Reply:
Its more like 0 to 1 Bar but the example gives out a negative number
![image](https://user-images.githubusercontent.com/73245684/110971092-ba403f00-835a-11eb-9f00-7f4148b35563.png)
![image](https://user-images.githubusercontent.com/73245684/110971175-d3e18680-835a-11eb-9b75-4ff0178029e4.png)

Reply:
ok